src/pages/ExamsPage/components/AllowanceListActions.jsx

```diff
@@ -10,8 +10,53 @@ import {
 import { DeleteOutline, EditOutline } from '@openedx/paragon/icons';
 import { useIntl } from '@edx/frontend-platform/i18n';
 
+import { Form } from 'react-router-dom';
+import { useState } from 'react';
 import messages from '../messages';
-import { useDeleteAllowance } from '../hooks';
+import { useDeleteAllowance, useEditAllowance } from '../hooks';
+
+// todo: add onEdit
+const EditModal = ({
+  isOpen, onCancel, onEdit,
+}) => {
+  const [ additionalTimeError, setAdditionalTimeError ] = useState(false);
```

Comment:
Should the Additional time only take in time in minutes or should it also handle time multiplier? Wasn't sure based on the figma here: https://www.figma.com/design/PXtvq1xV4R7dkVlLuqCE8s/Allowances?node-id=0-1&t=kWnGHiyO6EYb5OLP-0

Reply:
the multiplier isn't persisted so I think the only way to edit is change the raw value. The multiplier is mostly needed for doing multiple allowances at once.